src/wwx/2016/Justin Donaldson.md

```diff
@@ -0,0 +1,156 @@
+
+The biggest challenge overall has been how limited the Lua target is.  It lacks boolean operators, integers, and uses a shared convention for arrays and hash tables.  It indexes from 1, and scopes variables to functions, but does not auto-hoist them.  It also is incredibly fragmented, having significant incompatibilities between lua 5.2, luajit 5.2, and lua 5.3.  I'm trying to get 5.2 right (for lua and luajit) before tackling 5.3. So, only 5.2 is supported for now. 
+
+What has it been like working with the core team?
+The core team has been very supportive, but I tried hard not to add Lua specific problems to their burden for a release.  Only recently after crushing some particularly tricky bugs did I feel confident enough for general bug handling duties.
+
+Simn deserves special thanks, as he was the core team member that was the most active in discussions and development.  Nadako also gave a lot of good input.  I also wanted to thank PeyTy from the Haxe Community, who collaborated with me pretty heavily at the critical early phases.   
+
+What use case does the Lua target solve?
+Lua is particularly in vogue right now as a light weight scripting layer for more complex app logic.  The Haxe Lua target is designed to go everywhere that the Lua target can go.  From the speedy Luajit compiler, to the Torch machine learning framework, to the 
+LÖVE game engine, to the Corona cross-platform 2d mobile graphics SDK, to the WoW scripting client, to enumerable other editor and game plugin libraries,  the Haxe Lua target extends Haxe's flexible and powerful language features to those areas.
+
+I would be quick to add that Lua also provides safe means of sandboxing third party code.  For this reason, not all Haxe standard modules (e.g. sys/net) may be available in a given context due to platform limitations (e.g. WoW scripting).  In addition, many of these sys modules rely on third party lib support.  Accordingly, Lua is not as "batteries included" as many other Haxe targets, and may require some specific configuration that may or may not be possible. This is still an active area of development for the Lua target, and most of the current work for Lua involves development on these areas.  Stay tuned for more details!
+
+In closing, I wanted to say finishing the Lua target has been particularly satisfying. I came away more impressed with Nicholas, Hugh, Cauê, Simn, Franco, Frabbit, and the core target developers during the process.  In many ways they've tackled problems that were even more daunting than Lua.  I'm happy to have done something that puts me in that group.
```

Comment:
>I came away more impressed with Nicholas, Hugh, Cauê, Simn, Franco, Frabbit, and the core target developers during the process.  In many ways they've tackled problems that were even more daunting than Lua.

It would be interesting to hear more about this, maybe a "core" dev interview series or something :smile: 

>I wanted to say finishing the Lua target has been particularly satisfying.

Thanks for adding the 10th target, another target to investigate and compile too!